examples/chat/src/index.js

```diff
@@ -55,11 +55,11 @@ async function run() {
   const nodeDialer = new Node({
     peerId: idDialer,
     addresses: {
-      listen: ['/memory/test1']
+      // listen: ['/memory/test1']
```

Comment:
To avoid potential issues with listening and dialling only having one pair of streams, I commented this for now. The dialer node will not listen for connections. Once we progress, we should remove this and try both sides as described in the PR